tags/key.go

```diff
@@ -52,5 +51,5 @@ func (k *KeyString) String() string {
 	return fmt.Sprintf("%v", k.Name())
 }
 
-// CreateKeyString creates/retrieves the *KeyString identified by name.
+// CreateKeyString creates or retrieves a *KeyString identified by name.
```

Comment:
This creates or retrieves. I think we need a better name though. I am going to add a TODO for now.

Reply:
Yes. But this is not a real "New". If the a key with same name was already created, it will return the original one, not a newly created one. Arguably, create is a bad name also. Maybe "CreateOrRetrieve", "NewOrRetrieve"? I Couldn't think of a better name.
@rakyll  said she'll think of a better name in another comment.